src/sentry/snuba/discover.py

```diff
@@ -945,3 +947,141 @@ def get_facets(query, params, limit=10, referrer=None):
             )
 
     return results
+
+
+HistogramParams = namedtuple("HistogramParams", ["bucket_size", "start_offset", "multiplier"])
+
+
+def measurements_histogram_query(
+    measurements, user_query, params, num_buckets, precision=0, referrer=None,
+):
```

Comment:
Could we add a docstring to this function?

Reply:
Addressed in 27c9fbc